doc/source/validator.rst

```diff
@@ -0,0 +1,76 @@
+**********
+Validators
+**********
+
+Molecule is `opinionated`.  By being opinionated molecue tries to enforce a
+common way in which roles are maintained.
+
+Testinfra
+=========
+
+`Testinfra`_ is the default integration testing framework.  The tests are
+linted with `Flake8`_.  Flake8 supports storing its configuration in the
+project root as ``.flake8``.
+
+Usage
+-----
+
+.. code-block:: yaml
+
+    role_name/
+    ├── ...
+    └── tests/
+        └── test_*.py
+
+.. _`Testinfra`: http://testinfra.readthedocs.org
+.. _`Flake8`: http://flake8.pycqa.org/en/latest/
+
+Serverspec
+==========
+
+The tests are linted with `Rubocop`_.  `Rubocop`_ supports storing its
+configuration in the project root as ``.rubocop.yml``.
+
+Usage
+-----
+
+.. code-block:: yaml
+
+    role_name/
+    ├── ...
+    └── spec/
+        ├── spec_helper.rb
+        └── default_spec.rb
+
+When using serverspec, it's possible to target tests at the following levels:
+all instances, specific groups, specific instances.
+
+All files matching the pattern ``spec/*_spec.rb`` will be run against every
+instance.
+
+Tests located in ``spec/hosts/<hostname>/*_spec.rb`` will be run against the
+specific instance with the given hostname.
+
+Tests located in ``spec/groups/<groupname>/*_spec.rb`` will be run against the
+instances in the given group.
+
+Please note, this behavior only pertains to inventory generated by Molecule.
+Specifying outside inventory files or scripts will disable this functionality.
```

Comment:
Is this referencing the old static inventory functionality? I think we removed that, right?